### PR TITLE
Creation of a new admin debate details page - fixes #25

### LINKED ISSUE #25: Show debate details
The admin app should display debate details

### PR DESCRIPTION
**Pull request by AziXus**
New debate details page created

fixes #25

### CODE REVIEW

File: src/app/pages/admin/debate-questions/debate-questions.page.ts [hunk 1]
```diff
@@ -10,14 +10,13 @@ import {NotificationService} from "../../../services/notification.service";
   styleUrls: ['./debate-questions.page.scss'],
 })
 export class DebateQuestionsPage implements OnInit {
-
   availableQuestions  : any[] = [];
-  debateId            : string;
+  public debateId            : string;
```
Comment (BecaudA) [inline src/app/pages/admin/debate-questions/debate-questions.page.ts:14]: Strange alignement but ok.

File: src/app/pages/admin/debate-questions/debate-questions.page.ts [hunk 2]
```diff
@@ -10,14 +10,13 @@ import {NotificationService} from "../../../services/notification.service";
...
 
   constructor(
     private route         : ActivatedRoute,
     private router        : Router,
-    private debateManager : DebateService
+    private debateManager : DebateService,
```
Comment (BecaudA) [inline src/app/pages/admin/debate-questions/debate-questions.page.ts:19]: This `,` should not exist here I think.

File: src/app/pages/admin/debate-questions/debate-questions.page.ts (hunk 1 above)
Comment (AziXus) [reply src/app/pages/admin/debate-questions/debate-questions.page.ts]: Fixed alignement

File: src/app/pages/admin/debate-questions/debate-questions.page.ts (hunk 2 above)
Comment (AziXus) [reply src/app/pages/admin/debate-questions/debate-questions.page.ts]: Removed it.

**Pull request merged**